spec/index.html

```diff
@@ -279,13 +339,41 @@ <h3>Other Media Types</h3>
   <section id="n-triples-grammar">
     <h2>Grammar</h2>
     <p>An N-Triples document is a Unicode [[UNICODE]] character string encoded in UTF-8.
-    Unicode code points only in the range U+0 to U+10FFFF inclusive are allowed.</p>
-    <p>White space (tab <code>U+0009</code> or space <code>U+0020</code>) is used to separate two terminals which would otherwise be (mis-)recognized as one terminal. White space is significant in the production <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.</p>
-    <p>Comments in N-Triples take the form of '<code>#</code>', 
-      outside an <code>IRIREF</code> or <code>STRING_LITERAL_QUOTE</code>, and continue
-      up-to, and excluding, the end of line (<code>EOL</code>), 
-      or end of file if there is no end of line after the comment
-      marker. Comments are treated as white space.</p>
+      Unicode code points only in the range U+0 to U+10FFFF inclusive are allowed.</p>
+
+    <section id="sec-grammar-ws">
+      <h3>White Space</h3>
+
+      <p>White space (tab U+0009 or space U+0020) is allowed, but not required, before or after terminals.
+        Rule names below in capitals indicate where white space is significant;
+        these form a possible choice of terminals for constructing an N-Triples parser.</p>
```

Comment:
```suggestion
        Rule names below in capitals indicate where white space is significant.
</p>
```
Drop the "possible choice". The grammar actually says they are terminals.